packages/preview-middleware-client/types/sap.suite.ui.generic.template.d.ts

```diff
@@ -0,0 +1,11 @@
+declare module 'sap/suite/ui/generic/template/ListReport' {
+    import ManagedObject from 'sap/ui/base/ManagedObject';
+    export interface ListReportComponent extends ManagedObject {
```

Comment:
It should extend `sap.suite.ui.generic.template.lib.TemplateComponent` not ManagedObject.

Reply:
fixed